### ISSUE TITLE
generate the perl-Test-Quattor as part of the release

### ISSUE THREAD

**Issue by stdweird** [quattor/release#245]
maven-tools repo can now also produce an rpm with all the Test::Quattor modules
running the usual packaging method in the `package-build-scripts` does the trick.

**Comment by stdweird** [comment]
@jrha running the regular mvn package in the base of the repo fails, due to usual failure of build-profile submodule.
making this part of the release will allow people to run the unitttests without maven, something MS people would really like to do

**Comment by stdweird** [comment]
i added some jenkins magic, jenkins now provides rpm of latest master

https://jenkins1.ugent.be/view/Quattor/job/maven-tools/lastSuccessfulBuild/artifact/package-build-scripts/target/rpm/perl-Test-Quattor/RPMS/noarch/

**Comment by stdweird** [comment]
@ned21 i hope the jenkins url is enough until this is part of the release

**Comment by stdweird** [comment]
@ned21 i tagged this 16.8
@jrha this is already build by build_all_repos, but i guess it requires a bit more work to make it part of the release process?

**Comment by jrha** [comment]
Will test this during the RC phase.

**Comment by jrha** [comment]
So I've had a look during the 16.8 RCs and itt's not present by the time the RPMs are gathered during release. I haven't got any more time to look at it during this cycle, but I'll get back to it for 16.10.

**Comment by jrha** [comment]
@stdweird I'm just looking at this, what version numbering are we hoping for the RPM to have?
To match the main release version I would have to "release" the subdirectory of the maven tools repo, which might get confusing in tagging.

**Comment by stdweird** [comment]
@jrha what do you mean with main release version? 16.10 or 1.55? both are fine for me.

**Comment by jrha** [comment]
Exactly, either the version of the "Quattor release" (16.10) or the version of the maven tools (1.55), the question is which more correctly represents the contents of the RPM.

**Comment by stdweird** [comment]
in principle the maven tools version, but choose whatever is easiest. any rpm is better than no rpm.

**Comment by jrha** [comment]
Ok, in that case it should be done as part of the maven-tools release, which actually makes it slightly easier.
